src/main/java/ecdar/abstractions/BackendInstance.java

```diff
@@ -51,6 +53,14 @@ public void setDefault(boolean aDefault) {
         isDefault = aDefault;
     }
 
+    public boolean isThreadSafe() {
+        return isThreadSafe;
+    }
+
+    public void setIsThreadSafe(boolean aThreadSafe) {
```

Comment:
Why `aThreadSafe` and not `threadSafe`? I think above that they do `aDefault` because `default` is a reserved keyword?